yaml_files/cuba.yml

```diff
@@ -565,3 +555,59 @@ CUBA_KEYS:
     definition: Thermodynamic ensemble
     shape: [1]
     type: string
+
+  LATTICE_UC_ABC:
+    definition: Lattice translation vectors lengths (a, b, c) of an atomic unit cell
+    shape: [3]
+    type: double
+
+  LATTICE_UC_ANGLES:
```

Comment:
About your suggestion, we are not able to change this now, because it means deep changes in the wrapper/nCAD API. This CUBAs where defined long time ago (in the first moths of the projects) and implementation work with them.
Of course we can discuss if the name is the best one or it must be modified
In the next implementations we already implementing with your suggestions.